### DISCUSSION TITLE
No module named Ldm

### DISCUSSION THREAD

**Discussion by NullKizu** [continue-revolution/sd-webui-animatediff#550] [Q&A] [answered]
I am having a persistent problem when trying to install this in forge, I installed the extension just like any other user but just when starting stable diffusion the animateDiff option is not shown and that is because there is a script error because the ldm module is not found and I don't know how to install the module or find any solution. Any help is appreciated, thanks.

**Comment by ArnorWing** [comment, marked as answer]
There is a version of AnimateDiff jsut for FORGE
Check FORGE homepage
